pymeasure/display/widgets.py

```diff
@@ -146,9 +146,9 @@ def change_y_axis(self, axis):
 
 class TabWidget(object):
     """ Utility class to define default implementation for some basic methods """
-    def __init__(self, name, index):
+    def __init__(self, name, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Could you comment on why you allow passing `*args` and `**kwargs` to the `__init__` of `object`; if I'm not mistaken the `object` class doesn't allow passing any arguments.